### PR TITLE
shared DashBoards and Controller

### PR DESCRIPTION
**Pull request by Mazstar04**

### CODE REVIEW

File: Controllers/AuthController.cs [hunk 1]
```diff
@@ -115,17 +115,50 @@ public async Task<IActionResult> Login(LoginVM vm)
...
+                return RedirectToAction("Index", "Flock");
+            }
+            else if (role == "Farm Manager")
+            {
+                return RedirectToAction("Index", "Flock");
+            }
+            else
+            {
+
                 return NotFound();
```
Comment (laken11) [inline Controllers/AuthController.cs:162]: Return Unauthorized instead

File: Context/NikeDbContext.cs [hunk 2]
```diff
@@ -91,8 +91,6 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
            
             modelBuilder.Entity<FlockType>().Property(u => u.Name).IsRequired();
 
-            modelBuilder.Entity<FlockType>().Property(u => u.Description).IsRequired();
```
Comment (laken11) [inline Context/NikeDbContext.cs:94]: Why did remove this here?

**Review by laken11 — CHANGES_REQUESTED** [review]
Resolve comments

File: Context/NikeDbContext.cs (hunk 2 above)
Comment (laken11) [reply Context/NikeDbContext.cs:94]: You**

Comment (Mazstar04) [reply Context/NikeDbContext.cs:94]: You asked me too make d flocktype description optional

**Review by laken11 — APPROVED** [review]

**Review by laken11 — APPROVED** [review]

**Review by laken11 — APPROVED** [review]

**Review by laken11 — APPROVED** [review]

**Review by laken11 — APPROVED** [review]

**Pull request merged**